src/ocamlary/ocamlary.mli

```diff
@@ -17,82 +17,66 @@
 
 (** This is an {i interface} with {b all} of the {e module system} features.
     This documentation demonstrates:
-- comment formatting
-- unassociated comments
-- documentation sections
-- module system documentation including
- {ol
- {- submodules}
- {- module aliases}
- {- module types}
- {- module type aliases}
- {- modules with signatures}
- {- modules with aliased signatures}
-}
-
-A numbered list:
-+ 3
-+ 2
-+ 1
+    {ul
+     {- comment formatting }
+     {- unassociated comments }
+     {- documentation sections }
+     {- module
+        system
+        documentation
+        including
```

Comment:
This is strange doc formatting!

Reply:
Fixed: https://github.com/ocaml-ppx/ocamlformat/pull/2607